docs/reference/plugins.md

```diff
@@ -22,22 +22,48 @@ Siehe [MBMD](https://github.com/volkszaehler/mbmd) für ein Beispiel wie man Mod
 **Beispiel Lesen**:
 
 ```yaml
-source: mqtt
-topic: mbmd/sdm1-1/Power
-timeout: 30s # don't accept values older than timeout
-scale: 0.001 # floating point factor applied to result, e.g. for Wh to kWh conversion
+meters:
```

Comment:
M.E. gehört das hier nicht hin

Reply:
Ich finde schon. Siehe meine Begruendung in meinem Kommentar unten. tl;dr - Es ist ein Beispiel, dazu gehoert auch eine Hilfestellung *wo* man die section verwenden kann, insbesondere wenn es auf jede Einrueckung ankommt. Je mehr Kontext, umso besser.